### DISCUSSION TITLE
How to make AllowsTransparency = true for WPF/GTK host Window?

### DISCUSSION THREAD

**Discussion by DmitryBorodiy** [unoplatform/uno#16239] [Q&A]
Wpf project head in AppHead.xaml.cs
`
var window = (System.Windows.Window)Window.Current.GetNativeWindow();
window.Background = new SolidColorBrush(Colors.Transparent);
window.AllowsTransparency = true;
`

Hello, Uno Platform community!
I'm currently attempting to make my WPF host window transparent, but I've encountered an issue while using the AllowsTransparency and Background properties. The result is that the window appears to be entirely black. Additionally, I need to create a custom draggable area for the window. Acrylic backdrop also would be nice (WPF only). These requirements are specific to WPF/GTK.

![Screenshot 2024-04-11 190949](https://github.com/unoplatform/uno/assets/57910623/3f219c58-c982-4483-97a1-6d40c996749d)

**Comment by DmitryBorodiy** [comment]
https://github.com/unoplatform/uno/assets/57910623/b3e4d9b1-6ffa-4bcd-933b-3af40714ebb3

**Reply by MartinZikmund** [reply]
I'm suspecting this could be due to the Skia canvas we render on, but can't say for sure. Please create an issue with a simple repro and we will investigate
